virt_lightning/virt_lightning.py

```diff
@@ -816,9 +848,16 @@ def attachNetwork(self, network=None, nic_model=None, ipv4=None):
         disk_root.findall("./source")[0].attrib = {"network": network}
         disk_root.findall("./model")[0].attrib = {"type": nic_model}
 
-        if ipv4 and self.ipv4:
-            logger.error("ipv4 already set!")
-            exit(1)
+        if self.ipv4:
```

Comment:
I don't think we need to record the additional IPv4. The primary IP is special because we use it to reach the VM. But these additional IPv4 are already defined in the configuration file, I think this is a bit superflu.

So I feel like we should drop this part of change.